config/external.js

```diff
@@ -0,0 +1,24 @@
+const pkgInfo = require('../package.json');
+
+let external = ['dns', 'fs', 'path', 'url'];
+
+// add all dependencies defined in package.json into external array
+// as we don't want to bundle these
+if (pkgInfo.peerDependencies)
+  external.push(...Object.keys(pkgInfo.peerDependencies));
+if (pkgInfo.dependencies) external.push(...Object.keys(pkgInfo.dependencies));
```

Comment:
Do we need `optionalDependencies` too? These are generally considered dependencies if available.

Reply:
Sure, seems a reasonable thing to add.